tests/wpt/metadata/html/browsers/browsing-the-web/history-traversal/window-name-after-cross-origin-main-frame-navigation.sub.html.ini

```diff
@@ -1,3 +1,3 @@
 [window-name-after-cross-origin-main-frame-navigation.sub.html]
   type: testharness
-  expected: TIMEOUT
+  expected: ERROR
```

Comment:
The error here is actually a servers-side assertion, not a panic, so we went from timeout to fail one could say.